app/guid-file/controller.ts

```diff
@@ -24,6 +32,29 @@ export default class GuidFile extends Controller {
         return this.media.isTablet;
     }
 
+    // get versions from Waterbutler
+    @task
+    @waitFor
+    async loadRevisions() {
+        let response;
+        try {
+            response = await $.ajax({
+                type: 'GET',
+                url: `${this.model.links.download}?revisions=&`,
```

Comment:
So this is provided by WB, and doesn't paginate? Interesting.

Reply:
Yea, I tried a file with 26 revisions, and it seems that it doesn't paginate (at least at 26). I'm sure there are projects out there with files that have undergone many more revisions than that, so I may snoop/ask if there is indeed a point in which we need to worry about pagination